shap/utils/_legacy.py

```diff
@@ -97,19 +97,20 @@ def __init__(self, f, out_names):
         self.out_names = out_names
 
 
-def convert_to_model(val):
+def convert_to_model(val, keep_index=False):
```

Comment:
Suggest adding a docstring here to document the behaviour of `keep_index`

Reply:
Ok, a commit is added